Client/Frontend/Fakespot/Views/NoAnalysisCardView.swift

```diff
@@ -0,0 +1,153 @@
+        self.bodyLabelText = bodyLabelText
+        self.bodyLabelA11yId = bodyLabelA11yId
+        self.footerLabelText = footerLabelText
+        self.footerLabelA11yId = footerLabelA11yId
+    }
+}
+
+final class NoAnalysisCardView: UIView, ThemeApplicable {
+    private struct UX {
+        static let noAnalysisImageViewSize: CGFloat = 104
+        static let headlineLabelFontSize: CGFloat = 15
+        static let bodyLabelFontSize: CGFloat = 13
+        static let footerLabellFontSize: CGFloat = 13
+        static let contentStackViewSpacing: CGFloat = 8
+        static let contentStackViewPadding: CGFloat = 16
+    }
+
+    private var viewModel: NoAnalysisCardViewModel?
+
+    private lazy var cardContainer: ShadowCardView = .build()
+
+    private lazy var contentStackView: UIStackView = .build { stackView in
+        stackView.axis = .vertical
+        stackView.spacing = UX.contentStackViewSpacing
+    }
+
+    private lazy var noAnalysisImageView: UIImageView = .build { imageView in
+        imageView.image = UIImage(named: ImageIdentifiers.shoppingNoAnalysisImage)
+        imageView.contentMode = .scaleAspectFit
+    }
+
+    private lazy var headlineLabel: UILabel = .build { label in
+        label.numberOfLines = 0
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .headline,
+                                                            size: UX.headlineLabelFontSize,
+                                                            weight: .bold)
+    }
+
+    private lazy var bodyLabel: UILabel = .build { label in
+        label.numberOfLines = 0
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
+                                                            size: UX.bodyLabelFontSize)
+    }
+
+    private let footerLabel: UILabel = .build { label in
+        label.numberOfLines = 0
+        label.isUserInteractionEnabled = true
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
+                                                            size: UX.footerLabellFontSize)
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupLayout()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupLayout() {
+        let tapGesture = UITapGestureRecognizer(target: self,
+                                                action: #selector(didTapStartAnalysis))
+        footerLabel.addGestureRecognizer(tapGesture)
+        addSubview(cardContainer)
+        addSubview(contentStackView)
+        [noAnalysisImageView,
+         headlineLabel,
+         bodyLabel,
+         footerLabel].forEach(contentStackView.addArrangedSubview)
```

Comment:
nit, I think we can use `addSubviews(noAnalysisImageView, headlineLabel, bodyLabel, footerLabel)` ? A bit easier to read since it would align with the other methods

Reply:
Yes, please!